corehq/apps/sso/tasks.py

```diff
@@ -121,7 +121,7 @@ def send_idp_cert_expires_reminder_emails(num_days):
             )
 
 
-@periodic_task(run_every=crontab(minute=0, hour=2), acks_late=True)
+@periodic_task(run_every=crontab(minute=0, hour='*/4'), acks_late=True)
```

Comment:
do we know how long it takes for this task to run? Is this introducing a potentially heavy load on our celery machines?

Reply:
For CRS it takes less than 5 minutes (if I recall correctly) is it acceptable?

Reply:
Just tried. It takes 139 seconds for CRS. I think for now since we don't have many enterprise customer who turns on SSO, this is acceptable ?

Reply:
I think that's fine